crates/autopilot/src/driver_api.rs

```diff
@@ -1,5 +1,9 @@
 use {
-    crate::driver_model::{reveal, settle, solve},
+    crate::driver_model::{
+        reveal::{Reveal, Revealed},
```

Comment:
What's the reason for this rename?

I think it was a deliberate naming decision to have those be `solve::Request`, `solve::Response` etc. If you want to have different names in the context of this file you can import them `reveal::Request as Reveal`.

Reply:
No special reason, except I thought it is shorter and descriptive. Although, no strong preference here, reverted.